airflow/providers/docker/operators/docker.py

```diff
@@ -88,6 +100,9 @@ class DockerOperator(BaseOperator):
     :type tls_hostname: str or bool
     :param tls_ssl_version: Version of SSL to use when communicating with docker daemon.
     :type tls_ssl_version: str
+    :param mount_tmp_dir: Specify whether the temporary directory should be bind-mounted
+        from the host to the container
```

Comment:
Might be worth adding something like `Default is True` over here.

Reply:
```suggestion
        from the host to the container. Defaults to True
```